doc/manual/manual.tex

```diff
@@ -2268,10 +2268,152 @@ \subsection{Particles}
+describe all three options and start with a summary of their properties to
+guide users to an informed decision about the best option for their purpose.
 
-\subsection{System prerequisites}
+\begin{table}[htb]
+  \center
+  \begin{tabular}{|c|ccc|}
+    \hline
+    Feature & Compile \& Install & Virtual Machine & Docker Container \\
+    \hline
+    Speed overhead          & 0\%   & 30\%     & 0--5\%    \\
+    Disk overhead           & 0~GB  & 1~GB     & 200~MB       \\
+    Knowledge required      & Much  & Very Little & Little    \\
+    Root privileges required & No   & No (installed VM software) & Partially  \\
+    Embedded in native environment & Yes & No  & Partially    \\
+    MacOS support           & Yes   & Yes      & Yes    \\
+    Windows support         & No    & Yes      & Yes    \\
+    Local parallelization   & Yes   & Yes      & Yes            \\ 
+    Massively parallel computations & Yes & No & No \\ 
+    Modifying ASPECT        & possible & possible & possible \\
+    Configuring dependencies & possible & No   & No \\ \hline
+  \end{tabular}
+  \caption{Features of the different installation options of \aspect{}.}
+  \label{tab:install-options}
+\end{table}
+
+The available options can be best presented in form of typical use cases:
+
+\begin{enumerate}
+\item Virtual Machine (\aspect{} beginner and tutorial participant): The virtual machine image provides a fully prepared user environment that contains installations of \aspect{}, all required libraries, and visualization software on top of a full Linux environment. This way beginning users and tutorial participants can work in a unified  environment, thus minimizing installation time and technical problems. Due to the overhead of virtualizing a full operating system this installation typically needs more space, and is approximately 30~\% slower than a native installation. Additionally working in a virtual machine 'feels' differently from working in your usual desktop environment.
+
+\item Docker Container (advanced user with no need to configure / change the underlying libraries, possibly changing parts of \aspect): Docker containers are lightweight packages that only encapsulate the minimal dependencies to run an application like \aspect{} on top of the host operating system. They allow easy installation and usage of \aspect{} in a unified environment, while relying on the user's operating system to provide visualization software and model input data. When compared to the virtual machine it is simple to exchange files between the host operating system and the docker container, and it provides the benefit to work in the desktop environment you are used to. They have very little overhead in terms of memory and speed compared to virtual machines, and allow for reproducible computations. The container is set up with a standard \aspect{} installation, but this can be modified by advanced users (source code development within the container is possible).
```

Comment:
I removed the spaces. 
But you do not need to run a linux box. Docker is available for Mac and Windows as well. The minor difference is that on Windows Docker runs a minimal linux virtual machine that executes the containers. But the integration of this machine has become so effective that there is nearly no difference to running the containers natively. This is why I do not mention it, it would only confuse users.